native/src/test/java/io/ballerina/stdlib/http/transport/websocket/client/WriteTimeTestOutListener.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package io.ballerina.stdlib.http.transport.websocket.client;
+
+import io.ballerina.stdlib.http.transport.contract.websocket.WebSocketWriteTimeOutListener;
+
+/**
+ * A test listener for write idle timeout.
+ */
+public class WriteTimeTestOutListener implements WebSocketWriteTimeOutListener {
```

Comment:
Shall we rename this `WriteTimeTestOutListener ` to `WriteTimeTestOutTestListener`